pkg/kapp/cmd/app/apply_flags.go

```diff
@@ -55,6 +55,8 @@ func (s *ApplyFlags) SetWithDefaults(prefix string, defaults ApplyFlags, cmd *co
 
 	cmd.Flags().DurationVar(&s.WaitingChangesOpts.Timeout, prefix+"wait-timeout",
 		mustParseDuration("15m"), "Maximum amount of time to wait in wait phase")
+	cmd.Flags().DurationVar(&s.WaitingChangesOpts.ResourceWaitTimeout, prefix+"resource-wait-timeout",
```

Comment:
flag should be --wait-resource-timeout (all wait flags start with a wait prefix)

Reply:
Ah! got it. Done.